OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -63,6 +64,24 @@ public class CloakInfo : PausableConditionalTraitInfo
 		[Desc("The condition to grant to self while cloaked.")]
 		public readonly string CloakedCondition = null;
 
+		[Desc("Which image to use for effect when enter/exit cloak.")]
+		public readonly string EffectImage = null;
+
+		[Desc("Which sequence to use for effect when enter cloak.")]
+		[SequenceReference(nameof(EffectImage), allowNullImage: true)]
+		public readonly string EnterEffectSequence = null;
+
+		[Desc("Which sequence to use for effect when exit cloak..")]
+		[SequenceReference(nameof(EffectImage), allowNullImage: true)]
+		public readonly string ExitEffectSequence = null;
+
+		[PaletteReference(nameof(IsEffectPlayerPalette))]
+		public readonly string EffectPalette = "effect";
+		public readonly bool IsEffectPlayerPalette = false;
```

Comment:
```suggestion
		public readonly bool EffectPaletteIsPlayerPalette = false;
```

Reply:
Actually this seems unused so please remove it.

Reply:
third-party mod may need this on stealth effect

Reply:
fix the spelling, not removed